src/__fixtures__/default-schema.json

```diff
@@ -4,34 +4,40 @@
   "properties": {
     "name": {
       "type": "string",
+      "const": "Constant name",
+      "examples": ["Example name", "Different name"],
       "description": "The user's full name. This description can be long and should truncate once it reaches the end of the row. If it's not truncating then theres and issue that needs to be fixed. Help!"
     },
     "age": {
       "type": "number",
-      "minimum": 0,
-      "maximum": 150,
+      "minimum": 10,
+      "maximum": 40,
       "multipleOf": 10,
-      "exclusiveMinimum": true,
-      "exclusiveMaximum": true,
+      "default": 20,
+      "enum": [10, 20, 30, 40],
       "readOnly": true
     },
     "completed_at": {
       "type": "string",
       "format": "date-time",
-      "writeOnly": true
+      "writeOnly": true,
+      "pattern": "MM/dd/yyyy H:mm"
```

Comment:
I’m not sure what this is doing but pattern is meant to be regex. 

> pattern (This string SHOULD be a valid regular expression, according to the Ecma-262 Edition 5.1 regular expression dialect)

From OASv3.0.3.

Reply:
Right, changed to regex